src/index.css

```diff
@@ -9,6 +10,7 @@
   --font-body-heading: 600 36px Barlow;
```

Comment:
This was a mistake on my part. Can you change  `--font-body-heading: 600 36px Barlow` to  `--font-body-heading: 600 30px Barlow`